src/js/model.js

```diff
@@ -0,0 +1,38 @@
+import { LOTTO_RULE } from './constants.js';
+export default class Model {
+  #lottoList = [];
+
+  getLottoList() {
+    return this.#lottoList;
+  }
+
+  buyLotto(quantity) {
+    this.#lottoList = [];
+    for (let i = 0; i < quantity; i++) {
+      this.#lottoList.push(this.makeLottoNumbers());
+    }
+  }
+
+  makeLottoNumbers() {
+    return shuffle(makeAllLottoNumbers(LOTTO_RULE.MIN_NUMBER, LOTTO_RULE.MAX_NUMBER)).slice(
+      0,
+      LOTTO_RULE.NUMBERS_COUNT,
+    );
+  }
+}
+
+function makeAllLottoNumbers(min, max) {
+  const result = [];
+  for (let i = min; i <= max; i++) {
+    result.push(i);
+  }
+  return result;
+}
```

Comment:
```suggestion
function makeAllLottoNumbers(min, max) {
  if (min < 0) {
    throw new Error("~~")
  }
  return [...Array(max + 1).keys()].slice(min)
}
```
양수 범위만 생성하는 로또 같은 경우에는 이런 방식도 추천드려요!

Reply:
오.. 이런 방법이 있군요. 
감사합니다!!